app-drawer/app-drawer.html

```diff
@@ -170,6 +170,10 @@
 
       is: 'app-drawer',
 
+      hostAttributes: {
+        tabindex: '0'
+      },
+
```

Comment:
What about setting `tabindex="0"` on the `#contentContainer`?  If this is something we don't really want to expose to the user then it's better to not put it on the host.